### ISSUE TITLE
build fails: 'package code.google.com/p/log4go: unable to detect version control system for code.google.com/ path'

### ISSUE THREAD

**Issue by tmds** [inconshreveable/ngrok#310]
I am unable to build the code due to an error with the log4go package:

```
~/ngrok$ make
go fmt ngrok/...
GOOS="" GOARCH="" go get github.com/jteeuwen/go-bindata/go-bindata
bin/go-bindata -nomemcopy -pkg=assets -tags=debug \
		-debug=true \
		-o=src/ngrok/client/assets/assets_debug.go \
		assets/client/...
bin/go-bindata -nomemcopy -pkg=assets -tags=debug \
		-debug=true \
		-o=src/ngrok/server/assets/assets_debug.go \
		assets/server/...
go get -tags 'debug' -d -v ngrok/...
package code.google.com/p/log4go: unable to detect version control system for code.google.com/ path
github.com/gorilla/websocket (download)
github.com/inconshreveable/go-vhost (download)
github.com/inconshreveable/mousetrap (download)
github.com/nsf/termbox-go (download)
github.com/mattn/go-runewidth (download)
github.com/rcrowley/go-metrics (download)
Fetching https://gopkg.in/yaml.v1?go-get=1
Parsing meta tags from https://gopkg.in/yaml.v1?go-get=1 (status code 200)
get "gopkg.in/yaml.v1": found meta tag main.metaImport{Prefix:"gopkg.in/yaml.v1", VCS:"git", RepoRoot:"https://gopkg.in/yaml.v1"} at https://gopkg.in/yaml.v1?go-get=1
gopkg.in/yaml.v1 (download)
make: *** [deps] Error 1
```

**Comment by lukas2511** [comment]
code.google.com has been shut down completely by now

quick fix:
```
diff --git a/src/ngrok/log/logger.go b/src/ngrok/log/logger.go
index 41218ce..d25e83e 100644
--- a/src/ngrok/log/logger.go
+++ b/src/ngrok/log/logger.go
@@ -1,8 +1,8 @@
 package log

 import (
-       log "code.google.com/p/log4go"
        "fmt"
+       log "github.com/alecthomas/log4go"
 )

 var root log.Logger = make(log.Logger)
```

Not sure if that is the original project owner or just a fork, but it works.

**Comment by mamboer** [comment]
@lukas2511 thanks

**Comment by mart-e** [comment]
Hello,

Is it possible to make a new release with this patch? It has been fixed in February but is not included in 1.7 (which fails to build on my package manager because of this).
Thanks
